nixos/modules/config/xdg/mime.nix

```diff
@@ -32,14 +32,15 @@ in
       default = { };
       example = {
         "application/pdf" = "firefox.desktop";
-        "text/xml" = [
+        "text/*" = [
           "nvim.desktop"
           "codium.desktop"
         ];
       };
       description = ''
         Adds associations between mimetypes and applications. See the
         [specifications](https://specifications.freedesktop.org/mime-apps-spec/mime-apps-spec-latest.html#associations) for more information.
```

Comment:
Just noticed those are out of date. The new URL is https://specifications.freedesktop.org/mime-apps-spec/latest/associations.html